Resources/Prototypes/Nyanotrasen/psionicArtifacts.yml

```diff
@@ -1,21 +1,35 @@
-- type: weightedRandom
-  id: PsionicArtifactPool
-  weights:
-    ClothingHandsDispelGloves: 1
-    ClothingEyesTelegnosisSpectacles: 1
-    ClothingHandsGlovesColorYellowUnsulated: 1
-    PonderingOrbTelepathic: 1
-    ClothingShoesBootsMagBlinding: 0.5
-    LidOSaws: 0.25
-    BedsheetInvisibilityCloak: 0.15
-    # WeaponWandPolymorphCarp: 0.05
-    # WeaponWandPolymorphMonkey: 0.05
-    # WeaponWandFireball: 0.025
-    # WeaponWandDeath: 0.001
-    # WeaponWandPolymorphDoor: 0.05
-    SpawnSpellbook: 0.025
-    ForceWallSpellbook: 0.025
-    BlinkBook: 0.025
-    SmiteBook: 0.00025
-    KnockSpellbook: 0.025
-    FireballSpellbook: 0.01
+# Items to spawn when sacrificing a psionic
+- type: entityTable
+  id: PsionicSacrificeRewards
+  table: !type:AllSelector
+    children:
+    - id: MaterialBluespace1
+      amount: !type:RangeNumberSelector
+        range: 1, 4
+    - !type:NestedSelector
+      tableId: PsionicArtifacts
```

Comment:
comment out gamerloot and direction is happy

Reply:
all of it is gamer loot no

Reply:
Everything that isn't bluespace basically